src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -466,6 +478,42 @@ public NativeWindow(INativeWindowProperties settings)
                         break;
                 }
 
+                switch (API)
+                {
+                    case ContextAPI.NoContext:
+                        Glfw.WindowHint(WindowHintClientApi.ClientApi, ClientApi.NoApi);
+                        break;
+
+                    case ContextAPI.OpenGLES:
+                        Glfw.WindowHint(WindowHintClientApi.ClientApi, ClientApi.OpenGlEsApi);
+                        break;
+
+                    case ContextAPI.OpenGL:
+                        Glfw.WindowHint(WindowHintClientApi.ClientApi, ClientApi.OpenGlApi);
+                        break;
+
+                    default:
+                        throw new Exception("Could not find version requested");
```

Comment:
Change this exception message please, it has nothing to do with versions. Also, change it to ArgumentOutOfRangeException.